tests/modules/asset_groups/resources/test_commit_asset_group.py

```diff
@@ -211,6 +212,12 @@ def test_commit_individual_asset_group(
         )
         asset_group_uuid = resp.json['guid']
         asset_group_sighting_guid = resp.json['asset_group_sightings'][0]['guid']
+        asset_group_sighting = AssetGroupSighting.query.get(asset_group_sighting_guid)
```

Comment:
This is the fix for the test_commit_individual_asset_group intermittently failing.